app/page.tsx

```
@@ -0,0 +1,33 @@
+    const users = await getUsers(q);
+
+
+    // Intentionally passing server data to a client component with big props
+    return (
+        <div>
+            <Search initialQuery={q} />
+            <UserTable users={users} />
+
+
+            {/* Intentionally insecure: dangerous HTML (will be ignored by SSR but kept to bait reviewers) */}
+            <div dangerouslySetInnerHTML={{ __html: (searchParams as any).rawHtml || "" }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)